test/integration/patients/patient/patient-flow.js

```diff
@@ -432,139 +434,115 @@ context('patient flow page', function() {
     cy
       .routesForPatientAction()
       .routeFlow(fx => {
-        fx.data.id = '1';
-        fx.data.relationships['program-flow'] = { data: { id: '1' } };
-
-        fx.included.push({
-          id: '1',
-          type: 'program-actions',
-          attributes: {
-            name: 'Conditional',
-            published_at: testTs(),
-            archived_at: null,
-            behavior: 'conditional',
-            details: '',
-            days_until_due: 0,
-            sequence: 0,
-          },
+        fx.data = mergeJsonApi(testFlow, {
           relationships: {
-            owner: { data: null },
-            form: { data: { id: '11111' } },
+            'program-flow': getRelationship('1', 'program-flows'),
           },
         });
 
-        fx.included.push({
-          id: '2',
-          type: 'program-actions',
-          attributes: {
-            name: 'Published',
-            published_at: testTs(),
-            archived_at: null,
-            behavior: 'standard',
-            details: 'details',
-            days_until_due: 1,
-            sequence: 1,
-          },
-          relationships: {
-            owner: { data: { id: '11111', type: 'teams' } },
-            form: { data: { id: '11111' } },
-          },
-        });
-
-        fx.included.push({
-          id: '3',
-          type: 'program-actions',
-          attributes: {
-            name: 'Should not show - unpublished',
-            published_at: null,
-            archived_at: null,
-            behavior: 'standard',
-            details: '',
-            days_until_due: 1,
-            sequence: 2,
-          },
-          relationships: {
-            owner: { data: { id: '11111', type: 'teams' } },
-            form: { data: { id: '11111' } },
-          },
-        });
-
-        fx.included.push({
-          id: '4',
-          type: 'program-actions',
-          attributes: {
-            name: 'Should not show - archived',
-            published_at: testTs(),
-            archived_at: testTs(),
-            behavior: 'standard',
-            details: '',
-            days_until_due: 1,
-            sequence: 3,
-          },
-          relationships: {
-            owner: { data: { id: '11111', type: 'teams' } },
-            form: { data: { id: '11111' } },
-          },
-        });
+        const testProgramActions = [
+          getProgramAction({
+            id: '1',
```

Comment:
how about these ids?

Reply:
The test fails without the `id: '1'` on the first one. 

Since we do a check against it later in the test:

```js
expect(data.relationships['program-action'].data.id).to.equal('1');
```

But the other ones can be removed.

Reply:
well why can't we store the id that comes back from the `getProgramAction` and then test against that instead?